src/helpers/password.js

```diff
@@ -0,0 +1,21 @@
+const { randomBytes, scryptSync } = require('crypto')
+
+class Password {
+  static toHash(password) {
+    const salt = randomBytes(8).toString('hex')
+
+    const buffer = scryptSync(password, salt, 64)
+
+    return `${buffer.toString('hex')}.${salt}`
+  }
+
+  static compare(storedPassword, suppliedPassword) {
+    const [hashedPassword, salt] = storedPassword.split('.')
+
+    const buffer = scryptSync(suppliedPassword, salt, 64)
```

Comment:
synchronous password hashing is prone to timing attacks.  Would it be possible to switch `scryptSync` to async?

I get that this is just a demo, but it makes more since to demo something that can be turned into production code.